src/app.ts

```diff
@@ -404,6 +411,10 @@ export function newApp({
         API_CLIENT,
         new PecServerClientFactory(PECSERVERS)
       );
+      // Create the new messages service.
+      const NEW_MESSAGES_SERVICE = new NewMessagesService(
```

Comment:
APP_MESSAGES_SERVICE?

Reply:
Done in [d951fe2](https://github.com/pagopa/io-backend/pull/882/commits/d951fe237e764033c912db07a0b8f8252f376576)